lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -143,6 +158,31 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * @param {import('@csstools/media-query-list-parser').MediaQuery|import('@csstools/media-query-list-parser').GeneralEnclosed} node
+ * @param {Parameters<import('stylelint').CoreRules[ruleName]>[1]|undefined} secondaryOptions
```

Comment:
suggestion
non-blocking

maybe `/** @typedef {import('stylelint').CoreRules[ruleName]} Rule */` before l42
so that you can do `* @param {Parameters<Rule>[1]} [secondaryOptions]`

Reply:
I'm passing `secondaryOptions` to use `optionsMatches`, and I applied your suggestion to use `typedef`.